src/w3c/headers.js

```diff
@@ -248,7 +249,7 @@ export function run(conf) {
   }
   conf.licenseInfo = licenses[conf.license];
   conf.isCGBG = cgbg.includes(conf.specStatus);
-  conf.isCGFinal = conf.isCGBG && /G-FINAL$/.test(conf.specStatus);
+  conf.isCGFinal = conf.isCGBG && conf.specStatus.endsWith("G-FINAL");
   conf.isBasic = conf.specStatus === "base";
```

Comment:
Do we need this one given it’s in w3c/default.js?

Reply:
Hmm, maybe those duplicated fields can be removed. I'll try.

Reply:
I was to remove them from w3c/headers but instead removed from w3c/defaults, because the props are only used within headers.

Reply:
Ok, that makes sense.